x11-misc/bbweather/bbweather-0.6.3-r2.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="blackbox weather monitor"
+HOMEPAGE="https://www.netmeister.org/apps/bbweather/"
+SRC_URI="https://www.netmeister.org/apps/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="amd64 hppa ppc x86"
+IUSE=""
+
+DEPEND="dev-lang/perl
+	x11-libs/libX11"
+RDEPEND="${DEPEND}
+	net-misc/wget
+	x11-apps/xmessage
+	!<=x11-plugins/gkrellweather-2.0.7-r1"
+
+PATCHES=( "${FILESDIR}"/${PN}-asneeded.patch )
+
+src_prepare() {
+	gunzip doc/*.gz || die
+	sed -i \
+		-e "s:man_DATA:man1_MANS:;s:.gz::g;/^mandir/d" \
+		doc/Makefile.am || die
+	sed -i \
+		-e 's|-helvetica-|-*-|g' \
+		resource.cpp data/${PN}.{nobb,style} || die
+	eautoreconf
+	default
```

Comment:
You need to run `default` as first thing in `src_prepare`, 1) patch changes `configure.in`, 2) patches apply before files are touched by `sed`.